### ISSUE TITLE
Update JEFF 3.3 script with multi-tempertaure data

### ISSUE THREAD

**Issue by paulromano** [openmc-dev/data#28]
JEFF 3.3 used to distribute ACE files only at a single temperature, which is what our convert_jeff33.py script currently uses. It appears that they have files for [temperatures ranging from 293K up to 1800K](https://www.oecd-nea.org/dbdata/jeff/jeff33/index.html) now.

**Comment by Shimwell** [comment]
Interesting, do you think this is a chance to bring the two convert_Jeff scripts together

**Comment by paulromano** [comment]
Yeah, it's a possibility, although I'm not sure it would be cleaner than just having two separate scripts.

**Comment by Shimwell** [comment]
Looks like they have different temperature points between the two Jeff releases

Jeff 3.2 is 293, 400, 500, 600, 700, 800, 900, 1000, 1200, 1500, 1800
Jeff 3.3 is 293, 600, 900, 1200, 1500, 1800

This might require moving the list of temperatures into the release_details dictionary instead of the arguments where it currently is.
